engine/table/src/main/java/io/deephaven/engine/table/impl/SortListener.java

```diff
@@ -710,7 +710,7 @@ public void add(long numWrites, long numRequestedAdds) {
 
             // assert that the buffers are not overwritten
             Assert.eqTrue(writes.offer(numWrites), "writes.offer(numWrites)");
-            Assert.eqTrue(requestedAdds.add(numRequestedAdds), "requestedAdds.add(numRequestedAdds)");
+            Assert.eqTrue(requestedAdds.offer(numRequestedAdds), "requestedAdds.offer(numRequestedAdds)");
```

Comment:
This change has two purposes: to call attention to the only pre-`updateBy` usage of these data structures and because I think the code was inconsistent and should be changed.

Reply:
Discussed the impact of the change on this code w/ @rcaudy, agreed this change has zero impact.